.travis.yml

```diff
@@ -10,4 +10,11 @@ before_install: wget https://raw.github.com/splitbrain/dokuwiki-travis/master/tr
 install:
   - composer install
   - sh travis.sh
-script: cd _test && phpunit --stderr --group plugin_findologicxmlexport
+script:
+  - cd _test
+  - rm phpunit.xml
+  - wget https://raw.githubusercontent.com/thekeymaster/xmlexport-phpunit-tests/master/phpunit.xml
```

Comment:
Yes, but I did this in order to test, because there is no `phpunit.xml` in the master yet. On the last commit of this PR I will edit the `.travis.yml` accordingly and add the `phpunit.xml` to this branch. This also means that the last tests of this branch will probably fail.